cross/build-android-rootfs.sh

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env bash
```

Comment:
Thanks for the reminder; I had it in the issues checklist but forgot to merge pre-update. Will tackle this later today.